tests/e2e/e2e_evm_test.go

```diff
@@ -47,6 +73,291 @@ func (s *IntegrationTestSuite) testEVMQueries(jsonRCP string) {
 	})
 }
 
+// testEVM Tests EVM send and contract usage
+func (s *IntegrationTestSuite) testEVM(jsonRCP string) {
+	var (
+		err           error
+		valIdx        = 0
+		c             = s.chainA
+		chainEndpoint = fmt.Sprintf("http://%s", s.valResources[c.id][valIdx].GetHostPort("1317/tcp"))
+	)
+
+	// Get a funded EVM account and check balance transactions
+	key, _ := s.setupEVMwithFunds(jsonRCP, chainEndpoint, valIdx)
+
+	// Setup client
+	client, err := ethclient.Dial(jsonRCP)
+	s.Require().NoError(err)
+
+	// Deploy contract
+	s.Run("create and interact w/ contract", func() {
+		// Prepare auth
+		auth, err := bind.NewKeyedTransactorWithChainID(key, big.NewInt(1010))
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// Set optional params
+		auth.Value = big.NewInt(0)
+		auth.GasLimit = uint64(3000000) // gas limit
+		auth.GasPrice, _ = client.SuggestGasPrice(context.Background())
+
+		// Deploy
+		contractAddress, tx, counter, err := mock.DeployCounter(auth, client)
+		s.Require().NoError(err)
+
+		s.waitForTransaction(client, tx)
+		s.T().Logf("ContractAddress : %s", contractAddress.String())
+
+		// 6. Interact w/ contract and see changes
+		tx, err = counter.Increment(auth)
+		s.Require().NoError(err)
+		s.waitForTransaction(client, tx)
+
+		counterValue, err := counter.GetCounter(nil)
+		s.Require().NoError(err)
+		s.Require().Equal(big.NewInt(1), counterValue)
+	})
+}
+
+// testEVM Tests EVM send and contract usage
+func (s *IntegrationTestSuite) testERC20(jsonRCP string) {
```

Comment:
Since ERC20 is from a new module, can you move it to a different file and put it under a different testing procedure on e2e_test.go